File: week03/week03-1.py
# week 03-1.py 二合一python3 的版本
# leetcode 2579. count total number of colored cells
class Solution:
    def coloredCells(self, n: int) -> int:
        return n*n + (n-1)*(n-1)
        #觀察題目的圖片(不看英文、不看中文), 頭轉45度，看到正方形

File: week03/week03.-1.cpp
//week03.-1.cpp 二合一 c++的版本
//leetcode 2579. count total number of colored cells
class Solution {
public:
    long long coloredCells(int n) {
        long long ans = n;//要寫這行，把數字升級，變64位元[很長很長的整數]
        return ans*ans + (ans-1)*(ans-1); //c/c++/java/c# 都要加分號
    }
};
